Cognite.Simulator.Tests/UtilsTests/SimulationRunnerTest.cs

```diff
@@ -42,6 +41,7 @@ public async Task TestSimulationRunnerBase(bool useConstInputs)
         {
             var services = new ServiceCollection();
             services.AddCogniteTestClient();
+            services.AddLogger();
```

Comment:
this is being called here but also in `ConnectorBaseTest.cs` and in `AddCogniteTestClient`.
instead we can remove it from here and `ConnectorBaseTest` and change the order of calls in `AddCogniteTestClient`, so AddLogger comes after `services.AddSingleton( line 125`

Reply:
Right now all the log lines are duplicated due to this
```
05 16:06:14.389)
[2024-03-05 16:08:06.540 DBG] [Cognite.Simulator.Tests.UtilsTests.ConfigurationLibraryTest] Updating entity library. There are currently 2 entities. Extracted range: (2024-02-01 13:03:53.652, 2024-02-01 13:03:53.652)
[2024-03-05 16:08:06.540 DBG] [Cognite.Simulator.Tests.UtilsTests.ConfigurationLibraryTest] Updating entity library. There are currently 2 entities. Extracted range: (2024-02-01 13:03:53.652, 2024-02-01 13:03:53.652)
[2024-03-05 16:08:07.927 DBG] [Cognite.Simulator.Tests.UtilsTests.ModeLibraryTest] Updating file library. There are currently 2 files. Extracted range: (2024-03-05 16:06:14.260, 2024-03-05 16:06:14.389)
[2024-03-05 16:08:07.927 DBG] [Cognite.Simulator.Tests.UtilsTests.ModeLibraryTest] Updating file library. There are currently 2 files. Extracted range: (2024-03-05 16:06:14.260, 2024-03-05 16:06:14.389)

```